negotiation/message/diagram/contract-agreement-verification-message.puml

```diff
@@ -11,35 +11,14 @@ class "dspace:ContractAgreementVerificationMessage" {
     @type : "dspace:ContractAgreementVerificationMessage"
     dspace:providerPid : String
     dspace:consumerPid : String
-    cred:credentialSubject : dspace:ContractAgreementMessage
-    sec:proof : sec:Proof
+    dspace:referencedContractAgreementMessage : ReferencedMessage
 }
 
-class dspace:ContractAgreementMessage {
-    @type : "dspace:ContractAgreementMessage"
-    dspace:hash : String
+class ReferencedMessage {
```

Comment:
Referring to my previous comment: use another term like `HashedMessage`? `ReferencedMessage` is maybe not that "meaningful" considering the properties and purpose...